examples/lighting-app/nrfconnect/main/include/AppTask.h

```diff
@@ -74,6 +76,7 @@ class AppTask
 
     Function_t mFunction;
     bool mFunctionTimerActive;
+    bool mSoftwareUpdateEnabled;
```

Comment:
consider adding initializers for these variables.

My guess they go in bss and they are 0/false initialized by default, but it would feel better if I would see an explicit `= false` and `= nullptr`.